wiki/People/Beatmap_Nominators/General_Information/en.md

```diff
@@ -1,57 +1,49 @@
+## Important Links
 
-## Beatmap Nominator Icons
+- [Beatmap Nominator Rules](/wiki/Beatmap_Nominator_Rules) - which outlines both how to undertake various actions such as iconing maps, as well as how not to do things - you would want to avoid breaking these in any situation, so be sure to read them carefully! You can always ask a member of the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) if you are unsure about something, and we will be happy to help.
 
-Icons are the evidence of a Beatmap Nominator's efforts when checking maps, and though there are many on offer to you, these are the relevant ones to take note of:
+- [Code of Conduct for Mapping and Modding](/wiki/CoC) - As a Beatmap Nominator, you are expected to be an exemplary figure of the community, someone others look up to and respect, so misconduct may get you reprimanded or even expelled.
 
-### ![bubble icon](/wiki/shared/icon/bubble.gif) Bubble
+- [Ranking Criteria](/wiki/Ranking_Criteria) - If you are already a Beatmap Nominator, then you should be familiar with this. Essentially, it serves to explain what aspects of mapping are and are not allowed in the Ranked section, as well as including some general guidelines that people should try to follow in most cases. If you see a Qualified map breaking one of the rules, or feel that it could still use substantial improvement for whatever reason, feel free to report it in [this thread](https://osu.ppy.sh/community/forums/topics/447428), giving a brief outline of the problems you see.
 
-**This icon represents the first nomination.** As is with any icon on modding v1, the beatmapset must have at least 12 star priority before allowing you to place this. In Beatmap Discussions (Modding v2), the mapset must have at least 5 Hype and no currently unresolved Problem stamps. Make sure you're satisfied with all difficulties, even other game modes, before placing this. If the beatmapset was previously vetoed, you must also exhaustively explain why the reason for the veto should not prevent the map from moving forward.
+- For beatmaps with subjective quality concerns that don't necessarily break the rules and guidelines of the Ranking Criteria, but which you feel which make the beatmap in question unfit for the Ranked section, the [_beatmap veto_](/wiki/People/Beatmap_Nominators/Beatmap_Veto) allows you to withhold a beatmap from Qualification.
 
-### ![heart icon](/wiki/shared/icon/heart.gif) Heart
+- [Mapset Verifier](https://osu.ppy.sh/community/forums/topics/943895) - A tool built by NAT member [Naxess](https://osu.ppy.sh/users/8129817). This tool checks for a wide range of unrankable issues, as well as features a snapshot feature to easily compare versions of maps which is beneficial when rechecking. **This should not replace checking a beatmap yourself**, but instead be used as a tool to help double-check nothing slipped by.
```

Comment:
not sure if this is the right place to link community tools

Reply:
I think this is basically essential for BNs and would very strongly recommend tell any of them to use it, which is why its here.